### ISSUE TITLE
add tutorial

### ISSUE THREAD

**Issue by mmckerns** [uqfoundation/klepto#23]
add tutorial to cover major features; should be built white-paper style to demonstrate solving a problem or set of problems.  Good examples can be found in many of the tests.

**Comment by mmckerns** [comment]
A good start is here: https://github.com/mmckerns/tlkklp

**Comment by matkoniecz** [comment]
For start just simplest possible usage example in readme would be a great thing.

**Comment by mmckerns** [comment]
@matkoniecz: Good point.  I have simple examples in any of the above places I've noted, so I should do  what you suggest.

**Comment by nershman** [comment]
Even just adding that link to the README would be useful, most users will not see this issue.
